client/tsconfig.json

```diff
@@ -0,0 +1,53 @@
+{
+  "compilerOptions": {
+    // General Functionality
+    "module": "esnext",
+    "target": "esnext",
+    "moduleResolution": "node",
+    "importHelpers": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "skipLibCheck": true,
+
+    // Features
+    "allowJs": true,
+    "resolveJsonModule": true,
+    "useDefineForClassFields": true,
+    "lib": ["ESNext", "DOM", "WebWorker"],
+
+    // Code
+    "importsNotUsedAsValues": "error",
+    "isolatedModules": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+
+    // Emit (incremental)
+    "composite": true,
+    "declaration": true,
+    "declarationMap": true,
+    "emitDeclarationOnly": true,
+    "preserveWatchOutput": true
+  },
+
+  // Packages must be added here in order for their types to be compiled.
+  // Unfortunately, there isn't a decent way to do this automatically.
+  // The performance benefits are worth it, regardless.
+  "references": [
+    { "path": "./packages/wj-util" },
```

Comment:
I'm of the opinion that wikijump helper packages should be accessible through a vendor prefix `@wikijump`, so here `@wikijump/util`.

Reply:
If you mean aliases, I don't think in this instance we should do that as it adds some complications to module resolution. If you just mean scoping an NPM package, I think that would complicate the monorepo design a little bit? I can look into it.

Reply:
Ultimately as long as we have reasonable and consistent naming for wikijump-native modules/packages this should be fine.